packages/lesswrong/components/posts/PostsItem2.tsx

```diff
@@ -317,9 +339,13 @@ const PostsItem2 = ({
   // dismissRecommendation: If this is a Resume Reading suggestion, a callback
   // to dismiss it.
   dismissRecommendation,
+  draft,
```

Comment:
Why are you passing in draft as an argument, instead of getting it from the post? I'm looking to add this to the EAUsersProfile, but would have to pass it through a PostsList, but it seems a bit unnecessary.

Reply:
Fixed